File: src/main/java/net/hectus/neobb/turn/default_game/structure/glass_wall/TGlassWall.java
package net.hectus.neobb.turn.default_game.structure.glass_wall;

import net.hectus.neobb.player.NeoPlayer;
import net.hectus.neobb.structure.PlacedStructure;
import net.hectus.neobb.structure.Structure;
import net.hectus.neobb.structure.StructureManager;
import net.hectus.neobb.turn.default_game.attributes.clazz.NeutralClazz;
import org.bukkit.Material;
import org.bukkit.inventory.ItemStack;

import java.util.List;

public class TGlassWall extends GlassWallTurn implements NeutralClazz {
    public TGlassWall(NeoPlayer player) { super(player); }
    public TGlassWall(PlacedStructure data, NeoPlayer player) { super(data, player); }

    @Override
    public Structure referenceStructure() {
        return StructureManager.structure("glass_wall");
    }

    @Override
    public String color() {
        return "";
    }
}

File: src/main/kotlin/net/hectus/neobb/turn/default_game/structure/glass_wall/TGlassWall.kt
package net.hectus.neobb.turn.default_game.structure.glass_wall

import com.marcpg.libpg.storing.Cord
import net.hectus.neobb.player.NeoPlayer
import net.hectus.neobb.structure.PlacedStructure
import net.hectus.neobb.structure.Structure
import net.hectus.neobb.structure.StructureManager
import net.hectus.neobb.turn.default_game.attribute.clazz.NeutralClazz

class TGlassWall(data: PlacedStructure?, cord: Cord?, player: NeoPlayer?) : GlassWallTurn(data, cord, player), NeutralClazz {
    override val referenceStructure: Structure = StructureManager["glass_wall"]!!

    override fun color(): String = ""
}
